contracts/tests/ocr2.spec.ts

```diff
@@ -592,12 +594,55 @@ describe("ocr2", async () => {
         proposal: proposal.publicKey,
         receiver: owner.publicKey,
         authority: owner.publicKey,
+        tokenReceiver: recipientTokenAccount.address,
         tokenVault: tokenVault,
         vaultAuthority: vaultAuthority,
         tokenProgram: TOKEN_PROGRAM_ID,
       },
       remainingAccounts: payees,
     });
+  };
+
+  it("Initializes the OCR2 config", async () => {
+    await program.methods
+      .initialize(new BN(minAnswer), new BN(maxAnswer))
+      .accounts({
+        state: state.publicKey,
+        feed: feed.publicKey,
+        // payer: provider.wallet.publicKey,
+        owner: owner.publicKey,
+        tokenMint: token,
+        tokenVault: tokenVault,
+        vaultAuthority: vaultAuthority,
+        requesterAccessController: requesterAccessController.publicKey,
+        billingAccessController: billingAccessController.publicKey,
+        // rent: SYSVAR_RENT_PUBKEY,
+        // systemProgram: SystemProgram.programId,
+        // tokenProgram: TOKEN_PROGRAM_ID,
+        // associatedTokenProgram: ASSOCIATED_TOKEN_PROGRAM_ID,
+      })
+      .signers([state])
+      .preInstructions([
+        await program.account.state.createInstruction(state),
+        // await store.account.transmissions.createInstruction(transmissions, 8+192+8096*24),
+        // createFeed,
```

Comment:
?

Reply:
I hoped to create these in a single transaction but had some issues getting it to work.